adaptors/library/jobs/satusehat-create-observations.js

```diff
@@ -0,0 +1,140 @@
+// Create an observation in Satusehat
+
+fn(state => {
+  function createObservationData(item, state, result) {
+    const patientReference = `Patient/${state.patient.ihs_number}`;
+    const encounterReference = `Encounter/${state.encounterId}`;
+    const encounterDisplay = `${state.patient.full_name}'s ${state.visit.properties.visit_type} on ${state.visit.properties.visit_date}`;
+    return {
+      resourceType: 'Observation',
+      status: 'final',
+      category: [
+        {
+          coding: item.category,
+        },
+      ],
+      code: {
+        coding: item.code,
+      },
+      subject: {
+        reference: patientReference,
+      },
+      performer: [
+        {
+          reference: `Practitioner/N10000001`,
+          display: 'Voigt',
+        },
+      ],
+      encounter: {
+        reference: encounterReference,
+        display: encounterDisplay,
+      },
+      effectiveDateTime: state.visit.visit_date,
+      ...result,
+    };
+  }
+
+  /**
+   * Filters and maps observations based on the visit.
+   */
+  function mapObservations(state) {
```

Comment:
As `mapObserverations` is only called once, should this really be in its own function?